File: c#/task1.cs
using System;

class Task1 {
    static void Main() {
        Console.Write("Введите число: ");
        int n = int.Parse(Console.ReadLine());

        char start = (char)('A' + n - 1);

        for (int i = 1; i <= n; i++) {
            for (char c = start; c >= start - (i - 1); c--) {
                Console.Write(c);
            }
            Console.WriteLine();
        }
    }
}

File: c++/task1.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int N;
    cout << "Введите число: ";
    cin >> N;

    char start = 'A' + (N - 1); // Вычисляем начальный символ

    for (int i = 1; i <= N; i++) {
        string str = "";
        for (char j = start; j >= (start - (i - 1)); j--) {
            str += j;
        }
        cout << str << endl;
    }

    return 0;
}